iac-extensions/docker/src/test/java/org/sonar/iac/docker/parser/grammar/DockerLexicalGrammarTest.java

```diff
@@ -67,4 +67,17 @@ void testStringEol() {
       .matches("foo=bar")
     ;
   }
+
+  @Test
+  void shouldVerifyExecForm() {
+    Assertions.assertThat(DockerLexicalGrammar.EXEC_FORM)
+      .matches("[]")
+      .matches("[\"foo\"]")
+      .matches("[\"foo\", \"bar\"]")
+      .matches("[\"foo\",\"bar\"]")
+      .matches("[\"foo\" , \"bar\"]")
+      .notMatches("[\"foo\" \"bar\"]")
+      .notMatches("[\"foo\", \"bar\",]")
+      .notMatches("");
+  }
```

Comment:
Do we have to do this test? Is ExecFormTreeImplTest not enough?

Reply:
good catch, I will merge those